src/test/unit_tests/test_nb_native_fs.js

```diff
@@ -32,11 +33,28 @@ mocha.describe('nb_native fs', function() {
         });
     });
 
-    mocha.describe('lstat', function() {
-        mocha.it('works', async function() {
-            const path = 'package.json';
-            const res = await nb_native().fs.stat(DEFAULT_FS_CONFIG, path, { use_lstat: true });
-            const res2 = await fs.promises.stat(path);
+    mocha.describe('lstat', async function() {
+        const link_name = 'link.json';
+        const file_name = 'file.json';
+        const PATH = `/tmp/lstat${file_name}`;
+        const LINK_PATH = `/tmp/lstat${link_name}`;
+        mocha.before('lstat-before', async function() {
+            const fd = await fs.promises.open(PATH, 'w');
+            fd.close();
+            await fs.promises.symlink(PATH, LINK_PATH);
+        });
+
+        mocha.after('lstat-after', async function() {
+            await fs.promises.rm(PATH);
+            await fs.promises.unlink(LINK_PATH);
+        });
+
+        mocha.it('lstat on symbolic link', async function() {
+            const res = await nb_native().fs.stat(DEFAULT_FS_CONFIG, LINK_PATH, { use_lstat: true });
+            const res2 = await fs.promises.lstat(LINK_PATH);
+            // res3 is the stat of the regular file
+            const res3 = await fs.promises.stat(LINK_PATH);
```

Comment:
Maybe we can rename the variables (`res`, `res2`, `res3`) for readability? (in instead of or in addition to the comments).